plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerTypeMapping.java

```diff
@@ -378,6 +379,17 @@ public void testDate()
         }
     }
 
+    @Test
+    public void testDateJulianGregorianCalendarSwitch()
```

Comment:
is it redundant over `AbstractTestDistributedQueries`?

Reply:
It's basically redundant, but I put this because SQL Server connector's behavior isn't intuitive and we can't catch future database change if filtering the test case. I would remove if unnecessary.